oauth2-server/src/test/java/com/clouway/oauth2/PemKeyGenerator.java

```diff
@@ -0,0 +1,55 @@
+package com.clouway.oauth2;
+
+import com.clouway.oauth2.jws.Pem;
+import com.clouway.oauth2.jws.Pem.Block;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.util.Collections;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public class PemKeyGenerator {
+  private static KeyPair keyPair = getKeyPair(1024);
+
+  public static Pem.Block generatePublicKey() {
+    Pem pem = new Pem();
+    PublicKey publicKey = keyPair.getPublic();
+    String publicKeyPem = pem.format(new Block("PUBLIC KEY", Collections.<String, String>emptyMap(), publicKey.getEncoded()));
+    try {
+      return pem.parse(new ByteArrayInputStream(publicKeyPem.getBytes()));
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  public static Pem.Block generatePrivateKey() {
+    Pem pem = new Pem();
+    PrivateKey privateKey = keyPair.getPrivate();
+    String publicKeyPem = pem.format(new Block("PRIVATE KEY", Collections.<String, String>emptyMap(), privateKey.getEncoded()));
```

Comment:
privateKeyPem 


Actually you don't need format/parse, as you can return Pem.Block object directly. 
```java
new Pem.Block("PRIVATE KEY", Collections.<String, String>emptyMap(), privateKey.getEncoded());
```